### ISSUE TITLE
engauge.exe has stopped working

### ISSUE THREAD

**Issue by Clinewf** [markummitchell/engauge-digitizer#251]
I installed engauge 10.4 windows7-32 bit with the msi file on my 6 way Windows 7 Professional (8 Gb ram and lots of disk space). Windows is fully up to date. 

I can load a jpg of a simple one plot graph, select the coordinate points for the axes (3 points). When I try to select the curve I get the windows message: "engauge.exe has stopped working. A prolem caused the program to stop working correctly. Windows will close the program and .........

I've tried different graphs, used the wizard, read the tutorial rebooted Windows, and get the same response every time I try to make it work. I'm attached the graph file I've been working with. 

I keep assuming that I'm doing something stupid, but don't know what it is. All the other programs on this computer are working correctly -- python, brackets, Vensim, MS Office 2010, Thunderbird, LibreOffice, etc. I've been dealing with various versions of Windows for about the last 25  years and I'm stumped with this issue.
![155mm m107 cd-1k](https://user-images.githubusercontent.com/33990803/33235541-f2c89e92-d1ee-11e7-92e3-a083c580794a.jpg)

Any tips/help will be appreciated.
Cline Frasier

**Comment by markummitchell** [comment]
Sorry for the slow response.

If you select View / Background / Show Filtered Image do you see basically your original graph? After selecting the 3 axis points, are you using the blue button (Curve Point tool) to manually digitize, or are you using the green button (Segment Fill tool) and then clicking on a green-highlighted part of the curve - at which it sounds like the program crashes? I'd be happy to look at your dig file if you drag it into your next post. Also, you may try starting up in Safe mode to prevent any bad behavior by other software applications. You can google for the steps to start Safe mode. Finally, you can download a debug version of the program from https://ci.appveyor.com/project/markummitchell/engauge-digitizer/branch/master/job/vjwat6k969837yfe/artifacts. The download can be extracted using the 7-zip program. The debug version writes out a log file called 'Engauge.log' that you can drag into your next post. I can look at the log to maybe figure out the problem.

I experimented with your file but was unable to reproduce the problem.

**Comment by Clinewf** [comment]
![cfd and experimental a-n spinner 7 cal rocket](https://user-images.githubusercontent.com/33990803/33351385-686557ec-d458-11e7-9d75-81fea532ec3d.JPG)
I reran engauge with the graph file I sent previously, and got the following.
I do see the original graph. After selecting the axis points I got the red outline of the graph borders. Next, I used the green button. Then the form popped up that said: "Scanning" with the progress bar getting to 98%. Then Engauge quit with the Windows 7 msg: "engauge.exe has stopped working...." Clicking on the upper right X of that window exited engauge. At this point, there is no .dig file created. 

My next experiment was to open engauge and the same graph file and use the curve point tool. This worked perfectly. Then, I used some graph plots with colored lines and experimented with the color picker and segment fill tools (the files are attached). The good news is that engauge did not quit working. However, I could not get points created using either of tools. I experimented with the point match tool on the 155mm plot with my previous message and with file "CFD and experimental A-N...jpg" file attached to this note. engauge crashed using point match with both files.

My next experiment was to use Windows to uninstall engauge, search the registry for all keys containing "engauge" (there were 13) and then reinstall engauge. Still quits with the 155mm... file.

Then, I downloaded the debug file, unzipped it with 7zip and was unsuccessful running engauge. There didn't seem to be any installer so I just tried running the engauge.exe (3,865 KB dated 11/19/2017) from where it was extracted and nothing happened. Obviously, I'm missing something.

I haven't tried running in windows safe mode, primarily because every other program I have works fine.

For information, the regular download installed on my C: drive (302 GB free, Samsung SSD) and the data files were read from my F: drive (156 GB free, Samsung SSD). Of the 8 GB ram, task manager lists 3.5 GB free. Processor is a 6 way AMD Phenom II X6 1090T. I have dual monitors and operating system is Windows 7 Professional 64-bit SP1 fully up to date as of 11/21.

The two graphs mentioned above are attached. Hope all this helps sort out what may be going on.

Cline Frasier
![trajectory_width1](https://user-images.githubusercontent.com/33990803/33351358-3a9303e6-d458-11e7-8351-3087da33caee.JPG)

**Comment by markummitchell** [comment]
Not to pry, but your work seems like it may be for NASA. If so, it may be relevant that a few years back I had a support contract with that organization. They put me on a space shuttle contract so I could deliver a Windows version of Engauge. The process was very easy on my side, and may be on the other side given the great contracts people I've worked with.

**Comment by Clinewf** [comment]
I've now verified that Engauge works: 1) after Windows boots into SafeMode; 2) on my wife's HP Windows 7 with an AMD processor; and 3) on my Toshiba Windows 7 laptop -- with an Intel Core I7 processor.

There is something weird going on with my desktop system that Engauge fails when scanning for segments. It is specially weird because all my other graphics related programs, compilers, etc. all work fine. There is more information, including event logs, in the attached pdf. 

Since it appears that Engauge and some program on my system do not like each other, I'll do some experimenting over the next couple of days.
Cline
[Long message re engauge crashes.pdf](https://github.com/markummitchell/engauge-digitizer/files/1516633/Long.message.re.engauge.crashes.pdf)

**Comment by Clinewf** [comment]
I closed the issue in error.
Cline

**Comment by markummitchell** [comment]
I saw your notes. Interesting how malwarebytes interferes. I think you are right that something on your system is interfering with engauge. The standard releases do not write out a log file since it was rarely helpful and caused problems in some Windows and OSX installations. The 7z file you downloaded should write out a log file in the execution directory. Thanks for researching this issue so thoroughly!

**Comment by Clinewf** [comment]
[.engauge.log](https://github.com/markummitchell/engauge-digitizer/files/1523569/default.engauge.log)
I ran the debug version and loaded the same "155mm M107-1k.jpg" graph. I put one point on the far left side of the curve (at about 0.25 Mach and 0.13 Cd). Then I selected "segment." Scanning started and got to 98% when Engauge quit. The graph is attached for reference. The 
[engauge.log](https://github.com/markummitchell/engauge-digitizer/files/1523571/engauge.log)
.engauge.log file is also attached. Hope it is useful. (Ignore one of the log files. I thought the first one didn't download and so did it again.)

The above was with MalwareBytes unloaded and the software for my Brother laser printed not loaded at startup.

My next experiment will be to not load Acronis backup software at Windows startup and see if this makes a difference (Acronis buries itself deeply). I'm picking on Acronis because it is the only major piece of software that is unique to my desktop system.

Cline
![155mm m107 cd-1k](https://user-images.githubusercontent.com/33990803/33511816-b8d8f9dc-d6d7-11e7-9a16-4031bd5efe27.jpg)
